base/actions/actions.py

```diff
@@ -696,6 +695,10 @@ async def submit(
                         resp = await client.post(url, json=post_data, headers=headers)
                         # TODO: remove print
                         print(resp.content)
+
+                        template, group_arm = utils.get_display_message_template(resp)
+                        self.SLOTS.append(group_arm)
```

Comment:
why is this needed?

Reply:
No we don't need it, i was testing with it.